src/Setup.tsx

```diff
@@ -8,20 +8,19 @@ import {getSetup} from 'src/client'
 // Components
 import {ErrorHandling} from 'src/shared/decorators/errors'
 import PageSpinner from 'src/perf/components/PageSpinner'
-import {LoginPage} from 'src/onboarding/containers/LoginPage'
+import {CloudLoginPage} from 'src/onboarding/containers/CloudLoginPage'
 // lazy loading the signin component causes wasm issues
 import Signin from 'src/Signin'
 
 const OnboardingWizardPage = lazy(
   () => import('src/onboarding/containers/OnboardingWizardPage')
 )
-const SigninPage = lazy(() => import('src/onboarding/containers/SigninPage'))
+const OSSLoginPage = lazy(
+  () => import('src/onboarding/containers/OSSLoginPage')
+)
```

Comment:
Not for this PR - but what do you think about [this](https://reactjs.org/docs/code-splitting.html#named-exports) as far as resolving the issue with using named imports with React lazy? Adding a separate component for each lazy import is annoying, but maybe less so if we keep them in one folder (e.g. src/lazyimports).

Actually, having a separate wrapper component for anything that is imported using lazy could also give us space to [add retry functions around the components to deal with the chunk loading error issue.](https://medium.com/@botfather/react-loading-chunk-failed-error-88d0bb75b406)

Reply:
@wdoconnell we [have ways of handling named exports](https://github.com/influxdata/ui/blob/master/src/shared/containers/index.tsx#L102-L136) and react lazy loading. The idea about having a lazy component loader that handles exceptions might be worth looking into 🤔